b2share/modules/communities/api.py

```diff
@@ -85,10 +85,15 @@ def get(cls, id=None, name=None, with_deleted=False):
             raise CommunityDoesNotExistError(id) from e
 
     @classmethod
+    # TODO: change this into a search function, not just a list of communities
     # TODO: a query should be given
     def get_all(cls, start, stop):
         """Searches for matching communities."""
-        raise NotImplementedError()
+        start = int(start)
+        stop = int(stop)
+        from .models import Community as CommunityMetadata
+        metadata = CommunityMetadata.query.limit(stop)[start:]
```

Comment:
Even if this is a temporary fix, you might want to have a default sorting, on 'created' for example.